features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -50,6 +50,11 @@
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.avmfritz/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-bigassfan" description="Haiku Big Ass Fan Binding" version="${project.version}">
```

Comment:
As you refer to is as "Big Ass Fan Binding" at most places, you might want to remove the "Haiku" reference here - it might mess with the sorting and will make it more difficult for people to find it.